frontend/src/pages/WorkspaceLocked/WorkspaceLocked.styles.scss

```diff
@@ -1,16 +1,165 @@
-.workspace-locked-container {
-	text-align: center;
-	padding: 48px;
-	margin: 24px;
+$workspaceLocked: 'workspace-locked';
+$light-theme: 'lightMode';
+$ant-modal-override: 'ant-modal';
+$ant-tabs-override: 'ant-tabs';
+$ant-list-override: 'ant-list';
+
+@keyframes gradientFlow {
+	0% {
+		background-position: 0% 50%;
+	}
+	50% {
+		background-position: 100% 50%;
+	}
+	100% {
+		background-position: 0% 50%;
+	}
 }
 
-.workpace-locked-details {
-	width: 50%;
-	margin: 0 auto;
+// Unfortunately, we don't have a way to scope the mask since `.ant-modal-mask` is a parent node in the tree.
+// #TODO: We have to use the global selector to style the mask.
+.#{$ant-modal-override}-mask {
```

Comment:
make sure it doesn't affect anywhere else

Reply:
@vikrantgupta25, I’ve checked it locally, and it’s not causing any issues. In a worst-case scenario where this scope might affect another page, it’s just a modal backdrop blur. No functionality or visual should be impacted. I’ve considered this, which is why I added a #TODO for later.

Reply:
have you tried using `rootClassName` ? does it fall at the same level with the mask ?

Reply:
thank you for sharing this, this worked.